src/WebApp/flask/app.py

```diff
@@ -188,15 +188,47 @@ def operationalization_get_operation(operation, id = None):
             resp.headers['Content-type'] = 'application/json'
             return resp
     elif operation == 'services':
-        if id == None:        
+        if id == None:
             mm_response = model_management.get('services')
-            mm_response_json = json.loads(mm_response.text)    
+            mm_response_json = json.loads(mm_response.text)
+            config_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../App_Data/scoring.json'))
+            consumedId = ''
+            
+            if os.path.isfile(config_path):
+                with open(config_path, 'r') as f:
+                    scoring_config = json.loads(f.read())
+                    if 'id' in scoring_config:
+                        consumedId = scoring_config['id']
+                
+
+            for i in range(len(mm_response_json['value'])):
+                id = mm_response_json['value'][i]['id']
+                if consumedId == id:
+                    mm_response_json['value'][i]['consumed'] = "Yes" 
```

Comment:
True/False. Let's not re-invent Boolean.

Reply:
Sure. Will change that